kpi_dashboard/models/kpi_kpi.py

```diff
@@ -84,6 +88,27 @@ def write(self, vals):
             vals["value_last_update"] = fields.Datetime.now()
         return super().write(vals)
 
+    def _get_code_input_dict(self):
+        return {
+            "self": self,
+            "model": self.browse(),
+        }
+
+    def _forbidden_code(self):
+        return "(commit|rollback|getattr|execute)"
+
+    def _compute_value_code(self):
+        if re.search(self._forbidden_code(), (self.code or "").lower()):
+            raise ValidationError(_(
+                "The code cannot contain rollback or commit inside."
```

Comment:
Maybe 
```suggestion
                "The code cannot contain these terms '%s' inside." % self._forbidden_code()[1:-1].replace("|", ", ")
```

Reply:
I proposed another option. What do you think @bealdav ?